script/buildSQLGeom.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+"""
+This module will accept an input text file with (x,y,z) coordinates in a CSV-format list
+and generate a SQL string that will add these coordinates as a PostGIS Polygon
+to sites.geometry within a BETYdb instance.
+
+Command line usage:
+        -h      show help
+        -i      input filename
+        -e      EPSG reference number, default is 3857
+        -s      site_id in BETYdb sites table to associate geometry with
```

Comment:
could the `-s` argument accept either integer or string, and if a string, either query the site_id (if exists) or add a new site (if it doesn't exist)?